picamera2/previews/null_preview.py

```diff
@@ -16,13 +20,11 @@ def thread_func(self, picam2):
 
         sel = selectors.DefaultSelector()
         sel.register(picam2.camera_manager.event_fd, selectors.EVENT_READ, self.handle_request)
-        self.event.set()
+        self._started.set()
```

Comment:
Previously this was only ever `.set()`, so if this preview were restarted, the `.start()` method would roll forward as the event was set from the previous call to `.start()`